treeherder/extract/extract_jobs.py

```diff
@@ -12,15 +12,14 @@
 
 from treeherder.config.settings import REDIS_URL
 
-
 CONFIG_FILE = (File.new_instance(__file__).parent / "extract_jobs.json").abspath
 
 
 class ExtractJobs:
     def run(self, force=False, restart=False, start=None, merge=False):
         try:
             # SETUP LOGGING
-            settings = startup.read_settings(filename=CONFIG_FILE)
+            settings = startup.read_settings(filename=CONFIG_FILE, complain=False)
```

Comment:
`complain=False` will stop the emission of a not-useful warning message:

    WARNING: Ignoring arguments: ["extract_jobs", "--force"]

Reply:
Do you have any concerns that it could supress other important messages in the future?

Reply:
@armenzg  I am not concerned:  There are TWO systems looking at the command line parameters:  The Django management code is unaffected by this change; the second is the my extraction code.  This effectively turns off my code's error checking, since it is already checked by Django.